src/test/signalpathtest.h

```diff
@@ -265,10 +275,9 @@ class BaseSignalPathTest : public MixxxTest, SoundSourceProviderRegistration {
 
 class SignalPathTest : public BaseSignalPathTest {
   protected:
-    SignalPathTest() {
-        const QString kTrackLocationTest = QDir::currentPath() + "/src/test/sine-30.wav";
-        TrackPointer pTrack(Track::newTemporary(kTrackLocationTest));
-
+    SignalPathTest()
+            : BaseSignalPathTest() {
```

Comment:
Don't explicitly invoke the implicit default constructor of the base class. Useless noise.